modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase

```diff
@@ -518,6 +518,24 @@ definition {
 		Accounts.viewAssignee(assigneeName = "userfn");
 	}
 
+	@description = "This test adds an account with an avatar."
+	@priority = "3"
+	test CanAddAccountWithAvatar {
+		Accounts.openAccountsAdmin();
+
+		Accounts.addBusinessAccount(
+			accountLogo = "marketplace_icon.png",
+			accountName = "Business Account Name");
+
```

Comment:
Could you please add a step to confirm the uploaded file persist after clicking Done, thanks.